src/Microsoft.AspNetCore.Razor/CodeGenerators/Visitors/CSharpTagHelperFieldDeclarationVisitor.cs

```diff
@@ -53,8 +53,10 @@ protected override void Visit(TagHelperChunk chunk)
                     // Need to disable the warning "X is assigned to but never used." for the value buffer since
                     // whether it's used depends on how a TagHelper is used.
                     Writer.WritePragma("warning disable 0414");
-                    WritePrivateField(
-                        _tagHelperContext.TagHelperContentTypeName,
+                    Writer
+                        .Write("private ")
+                        .WriteVariableDeclaration(
+                        "string", 
```

Comment:
nit: Indent the parameters so it's clear they're being passed into the WriteVariableDeclartion method.